diff --git a/mod_form.php b/mod_form.php
index 700c247..89f3125 100644
--- a/mod_form.php
+++ b/mod_form.php
@@ -60,7 +60,6 @@ class mod_simplecertificate_mod_form extends moodleform_mod {
             get_string('certificatetext', 'simplecertificate'), null,
             simplecertificate_get_editor_options($this->context)
         );
-        #$mform->setType('certificatetext', PARAM_RAW);
         $mform->addRule('certificatetext', get_string('error'), 'required', null, 'client');
         $mform->addHelpButton('certificatetext', 'certificatetext', 'simplecertificate');
 
@@ -105,13 +104,12 @@ class mod_simplecertificate_mod_form extends moodleform_mod {
             get_string('secondimage', 'simplecertificate'), null,
             $this->get_filemanager_options_array());
         $mform->addHelpButton('secondimage', 'secondimage', 'simplecertificate');
-        $mform->disabledIf('secondimage', 'enablesecondpage', 'eq', 0);
+        // ...$mform->disabledIf('secondimage', 'enablesecondpage', 'eq', 0);.
 
         // Certificate secondText HTML editor.
         $mform->addElement('editor', 'secondpagetext',
             get_string('secondpagetext', 'simplecertificate'), null,
             simplecertificate_get_editor_options($this->context));
-        # $mform->setType('secondpagetext', PARAM_RAW);
         $mform->addHelpButton('secondpagetext', 'certificatetext', 'simplecertificate');
         $mform->disabledIf('secondpagetext', 'enablesecondpage', 'eq', 0);
 
